### PR TITLE
[12.0] ADD pos_multi_ean

### PR DESCRIPTION
**Pull request by eLBati**
In point of sale interface, allow to search products by multiple EAN (defined by product_multi_ean module) while scanning product barcode to add it to order.

### CODE REVIEW

File: pos_multi_ean/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "development_status": "Beta",
+    "category": "Point Of Sale",
+    "website": "https://github.com/OCA/pos",
+    "author": "TAKOBI, Odoo Community Association (OCA)",
+    "maintainers": ["eLBati"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto_install": True,
+    "preloadable": True,
```
Comment (legalsylvain) [inline pos_multi_ean/__manifest__.py]: why do you define the module as preloadable ?

**Review by legalsylvain — APPROVED** [review]
- code review - no test.
- One question. 

LGTM otherwise. 

Thanks !

File: pos_multi_ean/__manifest__.py (hunk 1 above)
Comment (eLBati) [reply pos_multi_ean/__manifest__.py]: :thinking: I copied from the template. It's more clear without it. Removed

**Review by ivantodorovich — COMMENTED** [review]
LGTM !

Regarding the module name, isn't `pos_product_multi_ean` more convenient, given that this is basically a glue module between `product_multi_ean` and `point_of_sale` ?

**Comment by eLBati** [comment]
@ivantodorovich I followed the same naming as `pos_supplierinfo_barcode`, considering that, in general, the module prefix should indicate the funcional area: prefix  `product` became `pos`.

No problem to change to `pos_product_multi_ean` if you guys think it's better

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Makes sense ! Thanks for the clarification

**Review by mrcast — APPROVED** [review]
Functional test ok!
Thanks

**Pull request merged**